jwst/datamodels/schemas/image.schema.yaml

```diff
@@ -7,6 +7,8 @@ allOf:
 - $ref: bunit.schema
 - $ref: photometry.schema
 - $ref: wcsinfo.schema
+- $ref: slitmeta.schema
```

Comment:
I don't think we want `slitmeta.schema` in `ImageModel`.

Reply:
In fact, we probably want the `resampmeta.schema` here, right?  We also want `resampmeta.schema` included in the `slitdata.schema` I believe?

Reply:
See my comment below: I don't think we need `resampmeta.schema` at all. The few entries in it can be moved to `core.schema`.

Reply:
I think we need `slitmeta.schema` included in `image.schema`, because there's at least one instance somewhere in the pipeline where a single instance of a spectroscopic slit gets resampled and was previously stored in a `DrizProductModel`. That was causing problems downstream, because `DrizProductModel` did not have slit-related keywords, like SLTNAME, which other code needed (that was the whole reason for this work to begin with).

I think it might the case where a NIRSpec fixed-slit exposure uses a subarray that only contains data from a single slit and that is/was getting resampled into a `DrizProductModel`, so presumably it would now get resampled into an `ImageModel`.

Reply:
Hmm, perhaps.  Maybe it should be resampled into a `SlitModel` instead?

Reply:
Currently in SlitDataModel, the attributes such as `source_type` are at the same level as `data`, `dq`, etc.  Is it the plan to retain this organization, or will `source_type`, etc., be moved under `meta`?

Reply:
I think that the plan for now was to simply replicate the existing structure of MultiSlit. It's then a longer-term goal to fix all such instances of data models that have a list of objects so that all meta attributes for each instance are in fact under the "meta" hierarchy. Of course that will require a lot of step code updates too.